.github/scripts/license/bam.license

```diff
@@ -0,0 +1,37 @@
+
```

Comment:
you may want to use the same license as the one used on the modules repositories

Reply:
* remove this license from a public repository !
* do not install business modules in OSS e2e tests, this adds a cyclic dependency